### DISCUSSION TITLE
Error when I write to stdout: how to solve it?

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#4047] [Q&A]
Hi,
if I run

```
quarto render ./report/4d9ceaa1-addd-43f0-a0a8-2623e818f348.qmd -o -
```

at the end I have, in the last line

```
</body></html>ERROR: NotFound: No such file or directory (os error 2)
```

If I add an output file, adding `>tmp.html` I have the file in the folder and `ERROR: NotFound: No such file or directory (os error 2)` message in the shell.

If I add `--metadata-file ../resources/only_content.yml` - it's a file to set `embed-resources: true` - the HTML output has no embedding.

How to solve it?

I'm using `1.3.115` in Debian 11.

Thank you

**Comment by cderv** [comment]
Thanks for the report. Probably an issue - This is tracked in https://github.com/quarto-dev/quarto-cli/issues/4052
